wpilibcExamples/src/main/cpp/examples/ElevatorSimulation/cpp/Robot.cpp

```diff
@@ -53,7 +73,15 @@ class Robot : public frc::TimedRobot {
   frc::DCMotor m_elevatorGearbox = frc::DCMotor::Vex775Pro(4);
 
   // Standard classes for controlling our elevator
-  frc2::PIDController m_controller{kElevatorKp, 0, 0};
+  frc::TrapezoidProfile<units::meters>::Constraints m_constraints{2.45_mps,
+                                                                  2.45_mps_sq};
+  frc::ProfiledPIDController<units::meters> m_controller{
+      kElevatorKp, kElevatorKi, kElevatorKd, m_constraints};
+
+  frc::ElevatorFeedforward feedforward(units::volt_t(kElevatorkS),
+                                       units::volt_t(kElevatorkG),
+                                       units::unit_t<kv_unit>(kElevatorkV),
+                                       units::unit_t<ka_unit>(kElevatorkA));
```

Comment:
If these are already in the right units, they can be passed directly

Reply:
When I tried that, it prompted me to pass in a type name + variable

Reply:
Well, it _should_ work.

Reply:
"error: 'kElevatorkA' is not a type"